src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let daysSpend = days * 40;
+
+  if (days >= 3 && days < 7) {
+    daysSpend -= 20;
+
+    return daysSpend;
```

Comment:
The `return` statement can be moved outside the conditional blocks to avoid repetition. Consider calculating `daysSpend` within the conditions and returning it once at the end of the function.